web/modules/custom/custom_module/custom_module.info.yml

```diff
@@ -0,0 +1,16 @@
+name: First Custom Module
+# Is it a module type or theme type that will be defined here.
```

Comment:
Looks good.
We can skip adding comments in .info.yml file
It's not required here.